### PR TITLE
fix(blocks): connectors resize and rotate

### PR DESCRIPTION
**Pull request by fundon**
Closes [BS-454](https://linear.app/affine-design/issue/BS-454/connector-resize-and-rotate)
Related to #6704

In the connectors scenario, just recalculate `source.position` and `target.position`.
Extract independent functions to resize elements.

> Maybe we can move these into their servies.

https://github.com/toeverything/blocksuite/assets/27926/ace30ee4-9b77-4962-8e49-8ada544ada63

### CODE REVIEW

**Comment by fundon** [comment]
* **#7256** <a href="https://app.graphite.dev/github/pr/toeverything/blocksuite/7256?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @fundon and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by golok727 — APPROVED** [review]

**Comment by Saul-Mirone** [comment]
@doouding PTAL

**Comment by doouding** [comment]
While the resizing is a good-to-have thing, I don't think rotation like this is necessary. Does the product team ask to add this explicitly?

**Comment by fundon** [comment]
It's not really rotation.

**Comment by fundon** [comment]
@Svaney-ssman @hwangdev97 PTAL

**Review by Svaney-ssman — APPROVED** [review]
GooGood

File: packages/blocks/src/root-block/edgeless/components/rects/edgeless-selected-rect.ts [hunk 1]
```diff
@@ -633,200 +886,51 @@ export class EdgelessSelectedRect extends WithDisposable(LitElement) {
...
-        bound.w = width * scale;
+        this.#adjustNote(element, bound, direction);
+        return;
+      }
 
-        height = bound.h / scale;
-        height = clamp(height, SYNCED_MIN_HEIGHT, Infinity);
-        bound.h = height * scale;
+      if (isEdgelessTextBlock(element)) {
+        this.#adjustEgelessText(element, bound, direction);
```
Comment (golok727) [inline packages/blocks/src/root-block/edgeless/components/rects/edgeless-selected-rect.ts:899]: Typo

**Pull request merged**